### PR TITLE
Update messaging sample to use horologist tile previews

### PR DESCRIPTION
**Pull request by ataulm**
Replace the TilePreviews that were in the sample with the ones from Horologist.

### CODE REVIEW

File: WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/ContactPreview.kt [hunk 1]
```diff
@@ -19,20 +19,25 @@ import android.graphics.BitmapFactory
...
 
 val emptyClickable = ModifiersBuilders.Clickable.Builder()
     .setOnClick(ActionBuilders.LoadAction.Builder().build())
     .setId("")
     .build()
 
+@Preview(
+    backgroundColor = 0xff000000, showBackground = true, widthDp = 100, heightDp = 100
+)
+annotation class IconSizePreview
```
Comment (ataulm) [inline WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/ContactPreview.kt:39]: Should this and :

```
import com.example.wear.tiles.tools.WearPreviewDevices
import com.example.wear.tiles.tools.WearPreviewFontSizes
import com.example.wear.tiles.tools.WearPreviewLocales
```

exist in Horologist?

Comment (yschimke) [reply WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/ContactPreview.kt:39]: Ideally yes, but https://issuetracker.google.com/issues/233511976

File: WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingTileRenderer.kt [hunk 2]
```diff
@@ -35,69 +36,56 @@ import androidx.wear.tiles.LayoutElementBuilders.VERTICAL_ALIGN_CENTER
...
                 imageResource
             )
         }
-
-        return resourcesBuilder.build()
     }
 
-    private fun RequestBuilders.ResourcesRequest.resourceRequested(resourceId: String) =
-        resourceIds.isEmpty() || resourceIds.contains(resourceId)
+    private fun imageResourceFrom(@DrawableRes resourceId: Int) = ImageResource.Builder()
```
Comment (yschimke) [inline WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingTileRenderer.kt:80]: Is this worth adding to Horologist? Either standalone function or an extension on Resources.Builder?

**Review by yschimke — APPROVED** [review]
Thanks, really good to see this used.

File: WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingTileRenderer.kt (hunk 2 above)
Comment (ataulm) [reply WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingTileRenderer.kt:80]: Yes, I added an issue for it :D https://github.com/google/horologist/issues/229

**Pull request merged**